src/ex03/cli.js

```diff
@@ -0,0 +1,128 @@
+import chalk from "chalk";
+import FileManager from "./FileManager.js";
+import Image from "ascii-art-image";
+import { Command } from "commander";
+import { PokemonClient } from "../ex1/scripts/Services/PokemonClient.mjs";
+import { Task } from "../ex1/scripts/Objects/Task.mjs";
+import { Pokemon } from "../ex1/scripts/Objects/Pokemon.js";
+
+const pokemonClient = new PokemonClient();
+const fileManager = new FileManager();
+
+function isOnlyNumbersAndCommas(userValue) {
```

Comment:
This should have been placed in a service file and not inside the CLI logic file